library/std/src/net/ip/tests.rs

```diff
@@ -313,6 +321,7 @@ fn ip_properties() {
     check!("ff08::", multicast);
     check!("ff0e::", global | multicast);
     check!("2001:db8:85a3::8a2e:370:7334", doc);
+    check!("2001:2::ac32:23ff:21", global | benchmarking);
```

Comment:
According to the [IANA IPv6 Special Address Registry](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml) benchmarking addresses are not globally reachable. However due to the current simplistic implementation of `Ipv6Addr::is_global` compared to `Ipv4Addr::is_global`, Rust currently considers benchmarking addresses global. I plan to address this in a future PR reworking `is_global`.

Reply:
The rework of `Ipv6Addr::is_global` has been submitted (#86634).